app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -988,7 +988,11 @@ DeviceDelegateOculusVR::RegisterImmersiveDisplay(ImmersiveDisplayPtr aDisplay) {
     return;
   }
 
-  m.immersiveDisplay->SetDeviceName("Oculus");
+  if (m.IsOculusQuest()) {
+    m.immersiveDisplay->SetDeviceName("Oculus Quest");
```

Comment:
to prepare for the [WebXR Device API](https://immersive-web.github.io/webxr/#applicationflow) (which intentionally does not expose identifier strings for devices), shouldn't we begin deprecating [`VRDisplay#displayName`](https://immersive-web.github.io/webvr/spec/1.1/#dom-vrdisplay-displayname)?

Reply:
It looks correct. Getting `displayName: "Oculus Quest"` from Oculus browser.

Reply:
:+1: good to know that we have parity with the Oculus Browser. in that case, we can defer deprecating the device string until WebXR is implemented (in both browsers) and web-compatibility isn't affected.